### DISCUSSION TITLE
DARK MODE

### DISCUSSION THREAD

**Discussion by michifuz1** [sumatrapdfreader/sumatrapdf#3693] [Ideas]
Hello, I'm new here, greetings everyone.
It is almost obvious that being a program for reading on screens it should have a dark mode, to protect your eyesight and greatly improve the readability of the texts.

Atte: Cristian Michel :)

**Comment by kjk** [comment]
It's already implemented in pre-release version: https://www.sumatrapdfreader.org/prerelease

Use menu `Settings / Theme` or command palette (`Cmd + K`) and `Select next theme` command.

**Reply by wishinghand** [reply]
Will this be implemented without inverting a PDF's colors?

**Reply by GitHubRulesOK** [reply]
@wishinghand 
there always has been a distinction between toggling colours (inversion, negation, reverse contrast) whatever.

Reversing colour is needed for reading blueprints or negatives as normal like viewing X-Rays so is not related to theming, thus then users reverse paper and ink that is not theming it is a canvas reversal of colour range.
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/d01e5b12-a576-46f4-b2c1-0f0f8df53aba)


Dark mode theme  is the User Interface the border data with a bright white PDF (or TXT) in the middle. 
This is Dark Theme
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/2d6256c3-7724-40c8-af46-5eedc800f528)
This is Darker Theme with inversion
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/05f7ce32-2a3c-40bb-afe4-e1c9959feca2)
Darker/darkest with a JPG
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/1773e192-341d-4712-ab3d-13e4a2135954)

**Comment by kjk** [comment]
@wishinghand You can de-invert by pressing `'i`. We set the inversion by default when switching to a dark theme but if you de-invert with `i` it will be remembered for the future

**Reply by Yafa0222** [reply]
And how do you do that. Can't seem to figure it out.

**Reply by GitHubRulesOK** [reply]
@Yafa0222 I do not know on all keyboards and it varied once  in test versions 
but in latest 3.5 pre rel https://www.sumatrapdfreader.org/prerelease it is the key between `u` and `o` on a qwerty keyboard
